### ISSUE TITLE
Feature request: include libtensorflow_io.so in the TensorFlow Serving Docker image

### ISSUE THREAD

**Issue by carlthome** [tensorflow/serving#1742]
For consuming various data formats (mp3 files in my case), I'd like to bundle relevant decoding ops in my SavedModel so there's no need for orchestrating additional frontend services with particular I/O logic, that subsequently calls my Serving deployment with huge tensors instead of byte strings.

In https://github.com/tensorflow/io there are a bunch of TensorFlow ops for consuming data, and I'd love to rely on them in my Serving deployment. Currently, I believe I would need to include the shared library in https://github.com/tensorflow/serving/blob/21971f2168bcaa527c35efce74a0acb2707af357/tensorflow_serving/model_servers/BUILD#L318 and compile Serving myself, but perhaps you already have plans for including the tensorflow-io ops in the Docker Hub images? Perhaps there could be alternate tags that includes more TensorFlow projects.

**TL;DR: It would be very convenient if the TensorFlow IO ops were made readily available in TensorFlow Serving.**

**Comment by shadowdragon89** [comment]
You are right. Now, you need to update the build file to include the needed ops and compile it.
We do not currently have plan to include the op into the binary by default.

**Comment by rmothukuru** [comment]
@carlthome,
Can you please respond to @shadowdragon89's comment. Thanks!
